mozilla_bitbar_devicepool/test_run_manager.py

```diff
@@ -139,6 +142,12 @@ def handle_queue(self, project_name, projects_config):
 
                             logger.info('test run {} started'.format(
                                 test_run['id']))
+                except RequestResponseError as e:
+                    # TODO: inspect message for
+                    #   "FileEntity with id 1508313 does not exist"
+                    logger.error("Test files have been archived. Exiting so configuration is rerun...")
+                    logger.error("%s: %s" % (e.__class__.__name__, e.message))
+                    sys.exit(1)
```

Comment:
We don't actually check if it is a 404 like we saw this morning or something else. Should we have a different kind of message for either case? I could see the case for something like

if error is because test file is missing:
   log.error exiting due to test file id {}  missing. Restart should recreate the missing files.
else
  log.error exiting due to an unexpected error.

It looks like the exception has both message and statusCode attributes. We should just go ahead and use them now.